examples/minetest/README.md

```diff
@@ -0,0 +1,11 @@
+# Minetest Game Server Example 
```

Comment:
Meta-question:

How many examples that use the same integration pattern do we want to maintain? Is this example more compelling than xonotic? supertuxcart? 

If we start to have N examples that do pretty much the same thing, then that increases the cost of upgrading them when we move to a newer version of golang, debian, etc. 

@markmandel

Reply:
I was going to come and ask exactly the same question. While this is cool, I'm not sure it's bringing anything of extra value to the table at this time -- so I was also debating the maintenance cost.

Maybe this would be better stored in a personal repo? And then we could add an "examples" section to https://agones.dev/site/docs/third-party-content/ ?

Reply:
Thank you for the review. 
I'll make a separate repo and file a separate PR for adding an examples section.

Reply:
I'm going to close this PR in favor of https://github.com/googleforgames/agones/pull/2258.